OracleManagementAgent/README.md

```diff
@@ -1,107 +1,129 @@
 # Oracle Management Agent Container Image
+
 This repository contains sample container configurations to facilitate installation and environment setup for DevOps users. This project includes Dockerfiles based on Oracle Linux and Oracle OpenJDK 8.
 
 The certification of Oracle Management Agent in a container does not require the use of any file presented in this
 repository. Customers and users are welcome to use them as starters, and customize/tweak, or create
 from scratch new scripts and Dockerfiles.
-
-
+<!-- markdownlint-disable MD013 -->
+**Note: The latest container files no longer require elevated privileges that were used to create a local system account in order to run the Oracle Management Agent and run the daemon process. The operator can now opt to create user accounts during image development (in Dockerfile) or at runtime they can simply provide the numeric UID and GID values that belong to a user found in the environment that hosts the container.**
+<!-- markdownlint-enable MD013 -->
 ## How to build and run
 
-Oracle Management Agent image uses the official `oraclelinux:7-slim` container image as the base image.
+Oracle Management Agent image uses the official `oraclelinux:8-slim` container image as the base image.
 
-#### Prerequisites
+## Prerequisites
 
 1. [Download the Management Agent software](https://cloud.oracle.com/macs)
 
     **Note: Select 'Downloads and Keys' then download 'Agent for LINUX (X86_64)' of the package type ZIP.**
 
-1. Copy the downloaded bundle to the same directory as the Dockerfile
+1. Copy the downloaded bundle to the same directory as the `Dockerfile`
 
     ```shell
+    # stage install bundle
```

Comment:
markdownlint has a rule to include output when you prefix a command with a '$' sign. Since cp has no output the only workaround I found was to add a useless comment.

Reply:
Remove the `$` and the useless comments. :)

Reply:
thanks, will remove